lib/lp_token_auth/core.rb

```diff
@@ -21,19 +22,22 @@ def issue_token(id, **payload)
         payload[:exp] = (Time.now + LpTokenAuth.config.get_option(:expires) * 60 * 60).to_i
       end
 
-      JWT.encode(
+      jwt = JWT.encode(
         payload,
         LpTokenAuth.config.get_option(:secret),
         LpTokenAuth.config.get_option(:algorithm)
       )
+
+      JWE.encrypt(jwt, private_key, enc: 'A256GCM')
```

Comment:
Would be nice is this was configurable via ENV var but defaults to `A256GCM`.

Reply:
We can add that

Reply:
Now we can configure the encryption type. The env variable is called `JWE_ENCRYPTION`.

Reply:
Could we also do this in the config file that gets made?